jgiven-html-app/src/js/util.js

```diff
@@ -2,6 +2,7 @@
  * Utility functions
  */
 
+let _ = require('lodash');
```

Comment:
please import 'lodash/core' to reduce the final artifact size. You can also use the ES6 syntax:

     import _ from 'lodash/core'